feature/experimental/policy/policy_vrf_selection/otg_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -132,8 +138,8 @@ func configInterfaceDUT(i *oc.Interface, me, peer *attrs.Attributes, subintfinde
 
 	if vlan != 0 {
 		// Add VLANs
-		singletag := s.GetOrCreateVlan().GetOrCreateMatch().GetOrCreateSingleTagged()
-		singletag.VlanId = ygot.Uint16(vlan)
+		singletag := s.GetOrCreateVlan()
```

Comment:
Why was this change needed? (I would not expect DUT configuration to change.)

Reply:
This was copied from the ATE test